### DISCUSSION TITLE
Use GPT-4 with ChatGPT-Writer

### DISCUSSION THREAD

**Discussion by EmilPoulsen** [GorvGoyl/ChatGPT-Writer#76] [Report Bugs and Issues]
Thanks for a fantastic tool.

I have a ChatGPT plus subscription to my OpenAI account, which makes GPT-4 available. However, when I use _ChatGPT-Writer_, it looks like it's still using GPT-3.5 to generate the responses. I can tell from inspecting the history when I log in to chat.openai.com.

Is there any way I can force _ChatGPT-Writer_ to use the GPT-4 model?

Thanks

**Comment by GorvGoyl** [comment]
extension should use the model available on ChatGPT site. Could you inspect
which model is used when using chatgpt site?

**Reply by EmilPoulsen** [reply]
@GorvGoyl On the ChatGPT site there is a dropdown that allows me to choose which model to use. It seems like _ChatGPT-Writer_ always uses GPT-3.

<img width="560" alt="image" src="https://user-images.githubusercontent.com/11620852/227870486-a1b2da75-7c04-4fb1-a1cf-be30310d0b9b.png">

**Reply by EmilPoulsen** [reply]
I noiced on [usechatgpt.ai](https://www.usechatgpt.ai/) (which _ChatGPT-Writer_ makes ads for) has a dropdown where the user can select which model to use. Perhaps this could be an option for _ChatGPT-Writer_ too? 
<img width="294" alt="image" src="https://user-images.githubusercontent.com/11620852/227871353-8a6e0ccf-6709-456e-9f96-67011bb70df5.png">

**Comment by GorvGoyl** [comment]
Thanks, this helps. will look into this on priority.

**Reply by EmilPoulsen** [reply]
No probs! Happy to help. Just ping me if you need help with testing

**Comment by KLucky13** [comment]
I also have a subscription and notice that emails are much better written if I let it generate by GPT4, not having the option to choose GPT 4 in ChatGPT-writer is a deal breaker for me.
